rsts/concepts/domains.rst

```diff
@@ -7,4 +7,5 @@ Domains provide an abstraction to isolate resources and feature configuration fo
 deployment environments. 
 
 For example: At Lyft, we develop and deploy Flyte workflows in development, staging, and production. We configure Flyte domains with those names, and specify lower resource limits on the development and staging domains than production domains. 
-We also use Domains to disable Launchplans and schedules from development and staging domains, since those features are typically meant for production deployments.
+
+We also use ``Domains`` to disable Launchplans and Schedules from development and staging domains, since those features are typically meant for production deployments.
```

Comment:
```suggestion
We also use ``Domains`` to disable launch plans and schedules from development and staging domains, since those features are typically meant for production deployments.
```